components/Team/TeamWorkList.tsx

```diff
@@ -1,42 +1,123 @@
 import { observer } from 'mobx-react';
-import { Ratio, Image, Accordion } from 'react-bootstrap';
+
+import Link from 'next/link';
+import {
+  Ratio,
+  Image,
+  Row,
+  Col,
+  Card,
+  Accordion,
+  Button,
+} from 'react-bootstrap';
 
 import { ScrollListProps, ScrollList } from '../ScrollList';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCalendarDay } from '@fortawesome/free-solid-svg-icons';
+
 import { TeamWork, TeamWorkType } from '../../models/Team';
 import activityStore from '../../models/Activity';
 
 export interface TeamWorkListProps extends ScrollListProps<TeamWork> {
   activity: string;
   team: string;
+  size?: 'sm' | 'lg';
+  onDelete?: (id: any) => void;
```

Comment:
```suggestion
  onDelete?: (id: TeamWork['id']) => any;
```
1. 直接从业务数据结构上引用类型即可
2. 回调函数返回值类型用 `any` 兼容性好